test/kpod_export.bats

```diff
@@ -3,11 +3,14 @@
 load helpers
 
 IMAGE="redis:alpine"
-
 function teardown() {
     cleanup_test
 }
 
+function setup() {
+    copy_images
```

Comment:
We only use `redis:alpine` in this file, so `copy_images` doesn't save us any time.